pkg/apis/networking/validation/validation.go

```diff
@@ -68,11 +68,21 @@ func ValidateNetworkPolicyPort(port *networking.NetworkPolicyPort, portPath *fie
 			for _, msg := range validation.IsValidPortNum(int(port.Port.IntVal)) {
 				allErrs = append(allErrs, field.Invalid(portPath.Child("port"), port.Port.IntVal, msg))
 			}
+			if port.EndPort != nil && *port.EndPort < port.Port.IntVal {
```

Comment:
```suggestion
			if port.EndPort != nil && *port.EndPort < port.Port.IntVal || *port.EndPort > 65535 {
```

Reply:
rather use validation.IsValidPortNum ?

Reply:
We can make this a follow up PR? For me no problem at all ;)

Reply:
@cpanato will move with the follow up :D